examples/acr/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM gcr.io/google-appengine/golang
```

Comment:
nit, you might want to use a smaller image to accelerate the build. (on GCR, we use `gcr.io/*` images because some of them are pre-pulled on the build node.)